### PR TITLE
remove note about selecting command-line environment

### PR DESCRIPTION
**Pull request by tdykstra**

### CODE REVIEW

File: aspnetcore/getting-started-1.1.md [hunk 1]
```diff
@@ -23,9 +23,6 @@ uid: getting-started-1.1
 
 1. Install [.NET Core](https://microsoft.com/net/core).
 
-   If you're on Windows, select the **Command line / other** environment.
```
Comment (Rick-Anderson) [inline aspnetcore/getting-started-1.1.md]: @danroth27  Can we change this to  
If you're on Windows and you don't want to install Visual Studio, select the **Command line / other** environment.

Installing VS is significant/unnecessary friction for someone who just wants a quick test run.

Comment (Rick-Anderson) [inline aspnetcore/getting-started-1.1.md]: We could even drop the image, but some warning might be helpful. New folks probably don't know they have the option of not installing VS.

Comment (danroth27) [inline aspnetcore/getting-started-1.1.md]: I think it's best to decouple ourselves from a installation experience for .NET Core. Any description or images that we provide for installing .NET Core runs the risk of going stale or out of date as the .NET site gets updated. All we should do is tell people to install .NET Core and provide a link.

Comment (scottaddie) [inline aspnetcore/getting-started-1.1.md]: @danroth27 Do you prefer that we refer to it as ".NET Core SDK" instead of ".NET Core"?

Comment (Rick-Anderson) [inline aspnetcore/getting-started-1.1.md]: @scottaddie  It should be ".NET Core SDK"  - right?

Comment (scottaddie) [inline aspnetcore/getting-started-1.1.md]: @Rick-Anderson That's how I refer to it. Just want to make sure we're all on the same page.

**Comment by Rick-Anderson** [comment]
@tdykstra  should we merge this?

**Comment by tdykstra** [comment]
Yes I'll have another one to make 2.0 the toc one and update install links

Sent from iPhone
________________________________

**Comment by Rick-Anderson** [comment]
@tdykstra  I was on the wrong page and merged this. Should I revert?

**Comment by tdykstra** [comment]
No it's ok as is

Sent from iPhone
________________________________

**Pull request merged**